docs/csharp/csharp-6.md

```diff
@@ -1,17 +1,570 @@
+The object produced from a string interpolation is a type that has an
+implicit conversion to either `System.String` or `System.IFormattableString`.
+
+The `FormattableString` type contains the format string, and the results
+of evaluating the arguments before converting them to strings. You can
+use public methods of `FormattableString` to specify the culture when
+formatting a string. For example, the following will produce a string
+using German as the language and culture. (It will use the ',' character
+to separate the integer portion from the decimal portion of a number,
+and the '.' character as the thousands separator.)
+
+```csharp
+FormattableString str = @"Average grade is {s.Grades.Average()}";
+var gradeStr = string.Format(null, 
+    System.Globalization.CultureInfo.CreateSpecificCulture("de-de"),
+    str.GetFormat(), str.GetArguments());
+```
+
+> [!NOTE]
+> The above example is not supported in .NET Core version 1.0.1. It is
+> only supported in the full .NET Framework.
+
+In general, string interpolation expressions produce strings as their
+output. However, when you want greater control over the culture used to
+format the string, you can specify a specific output.  If this is a capability
+you often need, you can create convenience methods, as extension methods,
+to enable easy formating with specific cultures.
+
+## Exception Filters
+
+Another new feature in C# 6 is ***exception filters***. Exception Filters
+are clauses that determine when a given catch clause should be applied.
+If the expression used for an exception filter evaluates to `true`, the
+catch clause performs its normal procesing on an exception. If the
+expression evaluates to `false`, then the `catch` clause is skipped.
+
+One use is to examine information about an exception to determine if a
+catch clause can process the exception:
+
+[!code-csharp[ExceptionFilter](../../samples/snippets/csharp/new-in-6/NetworkClient.cs#ExceptionFilter)]
+
+The code generated by exception filters provides better information about
+an exception that is thrown and not processed. Before exception filters
+were added to the language you would need to create code like the following:
+
+[!code-csharp[ExceptionFilterOld](../../samples/snippets/csharp/new-in-6/NetworkClient.cs#ExceptionFilterOld)]
+
+The point where the exception is thrown changes between these two examples.
+In the previous code, where a `throw` clause is used, any stack trace
+analysis or examination of crash dumps will show that the exception was
+thrown from the `throw` statement in your catch clause. The actual exception
+object will contain the original call stack, but all other information
+about any variables in the call stack between this throw point and the
+location of the original throw point has been lost. 
+
+Contrast that with how the code using an exception filter is processed:
+The exception filter expression evaluates to `false`. Execution never
```

Comment:
It shouldn't be a list. It's a series of cause and effect statements. See if the rewrite is more clear.